app/Controllers/HomeController.cs

```diff
@@ -14,6 +14,17 @@ public IActionResult Index()
             return View();
         }
 
+        [Route("~/sso")]
+        public IActionResult Sso()
+        {
+            ViewBag.AppUrl =
+                Request.Query["court"] == "coa"
+                    ? "/scjob/booking/coa/CaseSearch"
```

Comment:
This url was changed to "case-search" on develop, just a reminder 👍

Reply:
Thanks for the reminder